library/timing.c

```diff
@@ -527,4 +527,44 @@ int mbedtls_timing_self_test( int verbose )
 
 #endif /* MBEDTLS_SELF_TEST */
 
+#else
+volatile int mbedtls_timing_alarmed = 0;
```

Comment:
Not really related to this particular change, but volatile is insufficient to make this code correct on non-strongly ordered SMP targets (e.g. Arm).